docs/src/content/docs/tr/guides/i18n.mdx

```diff
@@ -0,0 +1,221 @@
+
+## Starlight'ın UI'sini Çevirin
+
+Starlight, çevrilen içerik dosyalarına ek olarak, varsayılan UI dizesini (örneğin, "Bu sayfada" içeriği) çevirmenize olanak tanır, böylece okuyucularınız seçilen dilde siteyi tamamen deneyimleyebilirler.
+
+Varsayılan olarak, Starlight İngilizce, Çekçe, Fransızca, Almanca, İtalyanca, Japonca, Portekizce, Felemenkçe, Danca, İspanyolca, Türkçe, Arapça, Norveççe, Farsça, İbranice, Basitleştirilmiş Çince, Korece, Endonezya, Rusça, İsveççe, Ukrayna, Vietnamca ve Galiçyaca çevrilen UI dizelerini içerir ve daha fazla varsayılan dil eklemek için [katkıda bulunmalara](https://github.com/withastro/starlight/blob/main/CONTRIBUTING.md) açıktır.
+
+Desteklediğiniz ek diller için çevirileri sağlayabilir veya varsayılan etiketlerimizi `i18n` veri koleksiyonu aracılığıyla geçersiz kılabilirsiniz.
+
+1. Eğer zaten yapılandırılmamışsa `src/content/config.ts` içinde `i18n` veri koleksiyonunu yapılandırın:
+
+   ```js
+   // src/content/config.ts
+   import { defineCollection } from 'astro:content';
+   import { docsSchema, i18nSchema } from '@astrojs/starlight/schema';
+
+   export const collections = {
+   	docs: defineCollection({ schema: docsSchema() }),
+   	i18n: defineCollection({ type: 'data', schema: i18nSchema() }),
+   };
+   ```
+
+2. Her ek dil için `src/content/i18n/` içinde bir JSON dosyası oluşturun. Örneğin, bu Arapça ve Basitleştirilmiş Çince için çeviri dosyalarını ekler:
+
+   <FileTree>
+
+   - src/
+     - content/
+       - i18n/
+         - ar.json
+         - zh-CN.json
+
+   </FileTree>
+
+3. Çevirmek istediğiniz anahtarlar için JSON dosyalarına çeviriler ekleyin. Sadece değerleri çevirin ve anahtarları İngilizce olarak bırakın (örneğin, "search.label": "Arama").
+
+   İşte Starlight'ın varsayılan İngilizce değerleri:
+
+   ```json
+   {
+   	"skipLink.label": "İçeriğe Git",
+   	"search.label": "Ara",
+   	"search.shortcutLabel": "(Aramak için / tuşuna basın)",
+   	"search.cancelLabel": "İptal",
+   	"search.devWarning": "Arama yalnızca üretim sürümlerinde kullanılabilir. \nYerel olarak test etmek için siteyi oluşturup önizlemeyi deneyin.",
+   	"themeSelect.accessibleLabel": "Tema seçin",
+   	"themeSelect.dark": "Koyu",
+   	"themeSelect.light": "Açık",
+   	"themeSelect.auto": "Açık",
+   	"languageSelect.accessibleLabel": "Dil seçin",
+   	"menuButton.accessibleLabel": "Menü",
+   	"sidebarNav.accessibleLabel": "Ana",
+   	"tableOfContents.onThisPage": "Bu sayfada",
+   	"tableOfContents.overview": "Genel Bakış",
+   	"i18n.untranslatedContent": "Bu içerik dilinizde henüz mevcut değil.",
+   	"page.editLink": "Sayfayı düzenle",
+   	"page.lastUpdated": "Son Güncelleme:",
+   	"page.previousLink": "Sonraki",
+   	"page.nextLink": "Önceki",
+   	"404.text": "Sayfa bulunamadı. URL yi kontrol edin veya arama çubuğunu deneyin."
+   }
```

Comment:
This should be left in English as indicated by the line immediately above this code block:

> İşte Starlight'ın varsayılan İngilizce değerleri:


```suggestion
   {
   	"skipLink.label": "Skip to content",
   	"search.label": "Search",
   	"search.shortcutLabel": "(Press / to Search)",
   	"search.cancelLabel": "Cancel",
   	"search.devWarning": "Search is only available in production builds. \nTry building and previewing the site to test it out locally.",
   	"themeSelect.accessibleLabel": "Select theme",
   	"themeSelect.dark": "Dark",
   	"themeSelect.light": "Light",
   	"themeSelect.auto": "Auto",
   	"languageSelect.accessibleLabel": "Select language",
   	"menuButton.accessibleLabel": "Menu",
   	"sidebarNav.accessibleLabel": "Main",
   	"tableOfContents.onThisPage": "On this page",
   	"tableOfContents.overview": "Overview",
   	"i18n.untranslatedContent": "This content is not available in your language yet.",
   	"page.editLink": "Edit page",
   	"page.lastUpdated": "Last updated:",
   	"page.previousLink": "Next",
   	"page.nextLink": "Previous",
   	"404.text": "Page not found. Check the URL or try using the search bar."
   }
```